dashboard/test/controllers/level_sources_controller_test.rb

```diff
@@ -196,41 +196,4 @@ def expect_s3_upload
       apple_mobile_web_app: true
     )
   end
-
-  test 'migrates old flappy levels' do
```

Comment:
This test is being deleted so as to not cause a merge conflict with another pending PR that deletes it. Really, I should deal with the merge, but given how many PRs I'm cranking out to fix these warnings, I'm too lazy.